test/tor-browser.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -ex
+# Just run the Tor Browser with HTTPS Everywhere
+
+# Get to the repo root directory, even when we're symlinked as a hook.
+
+if [ -n "$GIT_DIR" ]
+then
+    # $GIT_DIR is set, so we're running as a hook.
+    cd $GIT_DIR
+else
+    # Git command exists? Cool, let's CD to the right place.
+    git rev-parse && cd "$(git rev-parse --show-toplevel)"
+fi
+
+
+source utils/mktemp.sh
```

Comment:
It just makes `mktemp` available across `GNU/Linux & BSD` systems, apparently.  This is documented in the `utils/mktemp.sh` file.